app/services/data/auto-approve-claim-expenses.js

```diff
@@ -0,0 +1,21 @@
+const config = require('../../../knexfile').asyncworker
+const knex = require('knex')(config)
+
+const statusEnum = require('../../constants/status-enum')
+
+module.exports = function (claimExpenses) {
+  var updates = []
```

Comment:
- select all ClaimExpense.ClaimExpenseId where ClaimId
  - build promise array calling a function that takes ClaimExpenseId which selects then updates to set ApprovedCost/Status